apps/regression/base.hu.i18n

```diff
@@ -1,21 +1,20 @@
 RegressionApp = "Regresszió"
 RegressionAppCapital = "REGRESSZIÓ"
 Regression = "Regresszió"
-Reg = "reg"
 MeanDot = "Középérték"
 RegressionCurve = "Regressziós görbe"
-XPrediction = "Jóslás megadva X"
+XPrediction = "Jóslás X megadva"
 YPrediction = "Jóslás Y megadva"
 ValueNotReachedByRegression = "Az ablakban az érték még nem volt elérve"
 NumberOfDots = "Pontok száma"
 Covariance = "Kovariancia"
 Linear = "Lineáris"
-Proportional = "Proportional"
+Proportional = "Általános"

```

Comment:
Proportional is technically "Arányos" in Hungarian altough when speaking about a functions and such it should be "egyenes arányosság" but i think "Arányos" is enough and everyone understands.